src/coreclr/inc/pedecoder.h

```diff
@@ -81,6 +81,8 @@ inline CHECK CheckOverflow(RVA value1, COUNT_T value2)
 #define IMAGE_FILE_MACHINE_NATIVE   IMAGE_FILE_MACHINE_ARMNT
 #elif defined(TARGET_ARM64)
 #define IMAGE_FILE_MACHINE_NATIVE   IMAGE_FILE_MACHINE_ARM64
+#elif defined(TARGET_S390X)
+#define IMAGE_FILE_MACHINE_NATIVE   IMAGE_FILE_MACHINE_UNKNOWN
```

Comment:
I would create an IMAGE_FILE_MACHINE_S390X instead. I would define it as 0x0390 in pal.h. @jkotas any objections on this?

Reply:
This sort of concerns me because it is a PE concept. I realize this is for Linux but it is still a [PE flag](https://docs.microsoft.com/windows/win32/debug/pe-format). `0x0390` is not set at present I'm just making sure we don't create a situation we may regret.

Reply:
In the discussion here https://github.com/dotnet/runtime/issues/52909#issuecomment-844338593, the conclusion also was that we shouldn't need to extend the PE file format.

Maybe we can defer the decision until such time we actually see a specific need for PE format extensions, if any?

Reply:
Ok, makes sense.